packages/design-system/src/components/UsaBanner/__snapshots__/UsaBanner.test.jsx.snap

```diff
@@ -11,20 +11,13 @@ exports[`UsaBanner renders correctly 1`] = `
     >
       <svg
```

Comment:
The USA Banner component is where I'm unsure how the icons should behave

Reply:
My thinking here:

- The US flag is visually significant, so it should have a title.
- The down chevron is decorative and should not have a title, as the accessible name and ARIA describe functionality
- The circle icons are decorative, as the subheadings and copy describe the functionality
- The lock icon is visually significant, so it should have a title, and that title should be consistent across the whole site where the lock icon is used in this context.

Reply:
Thanks for your input! I updated accordingly